x/campaign/types/genesis_test.go

```diff
@@ -306,3 +309,43 @@ func TestGenesisState_Validate(t *testing.T) {
 		})
 	}
 }
+
+func TestGenesisState_ValidateParams(t *testing.T) {
```

Comment:
Params validaty test cases should be complete in the `types/params_test.go`. For genesis state we assume `params.ValdiateBasics()` is fully tested and therefore we should just test once that the genesis state is invalid if the params are invalid

Reply:
I see `lanch/types/genesis_test.go` and I should just remove the last test case (the valid one) to make it consistent with `TestGenesisState_ValidateParams`. Is this what you are asking?

Reply:
I'm thinking about two test cases:
- Genesis state is valid
- Genesis state is invalid because params are invalid (whatever the reason is)

Reply:
Yes good point, we added more test case than necessary. I believe we can change this in https://github.com/tendermint/spn/issues/525 if we can consider this issue as a general refactoring of the `launch param`